### DISCUSSION TITLE
Livewire not working on Laravel8 after Laravel project updated to Laravel8 from Laravel 5.8

### DISCUSSION THREAD

**Discussion by TheAbhishekIN** [livewire/livewire#5058] [1. Help]
- Laravel Version: 8.83.17
- PHP Version: 7.4.19

### Description:

- I updated my Laravel 5.8 project to Laravel 8.
- after that on Laravel 8, Livewire was installed and it was installed via composer without any error.
- Still, after installing when I tried to make a component it gave `Command "make:livewire" is not defined`. error
- and on publishing assets `php artisan livewire:publish --config` it gives` There are no commands defined in the "livewire" namespace.` this error on console
- also @livewireStyles and @livewireScripts not outputting anything

Thank You

**Comment by joshhanley** [comment]
@TheAbhishekIN hmm that is strange! Maybe check that your composer.json file has the required scripts

https://github.com/laravel/laravel/blob/52863d9e4aa41f63592e8c98e5fe717ee7f06a18/composer.json#L35-L49

Hope this helps!
